---
title: '&#39;Při chybě&#39; nejsou platné uvnitř příkazů &#39;použití&#39; příkazy'
ms.date: 07/20/2015
f1_keywords:
- vbc36013
- bc36013
helpviewer_keywords:
- BC36013
ms.assetid: 5b399bf9-6595-46e0-a808-378f6c28568b
ms.openlocfilehash: cdcd4d5797698d7fcec134a0dacc146c199ec4f1
ms.sourcegitcommit: 2eceb05f1a5bb261291a1f6a91c5153727ac1c19
ms.translationtype: MT
ms.contentlocale: cs-CZ
ms.lasthandoff: 09/04/2018
ms.locfileid: "43504493"
---
# <a name="39on-error39-statements-are-not-valid-within-39using39-statements"></a>&#39;Při chybě&#39; nejsou platné uvnitř příkazů &#39;použití&#39; příkazy
`On Error` Příkazu se zobrazí v rámci `Using` příkaz ale není v tomto kontextu platné.  
  
 **ID chyby:** BC36013  
  
## <a name="to-correct-this-error"></a>Oprava této chyby  
  
-   Použití strukturovaného zpracování, jako například chyby `Try…Catch` bloku, v místě `On Error` příkaz.  
  
## <a name="see-also"></a>Viz také  
   
 [Kdy použít výjimky strukturovaných a nestrukturovaných zpracování (Visual Basic)](https://msdn.microsoft.com/library/e897d7ca-07e8-45dd-8a6d-a5b2a2fc9b9a)  
 [Příkaz On Error](../../visual-basic/language-reference/statements/on-error-statement.md)  
 [Postupy: testování kódu pomocí bloku Try... Blok catch v jazyce Visual Basic](https://msdn.microsoft.com/library/8368e205-ed73-4185-a247-af84fb4fafa9)  
 [Příkaz Try...Catch...Finally](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
